main.tf

```diff
@@ -89,6 +90,7 @@ module "pttp-infrastructure-ci-pipeline-dns-dhcp" {
   github_repo_name         = "staff-device-dns-dhcp-infrastructure"
   git_branch_name          = "main"
 
+  name        = "${module.label.id}-dns-dhcp-core-pipeline"
```

Comment:
Intermediate step to preserve other pipeline names

Reply:
fairs, didn't see this during my comment